src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -405,7 +405,7 @@ public void BeginBuild(BuildParameters parameters)
 
                 if (BuildParameters.DumpOpportunisticInternStats)
                 {
-                    OpportunisticIntern.Instance.EnableStatisticsGathering();
+                    StringTools.StringTools.EnableDiagnostics();
```

Comment:
nit: import StringTools namespace.

Reply:
Done. Also renamed the `StringTools` class because a type named `StringTools` in a namespace named `StringTools` is only causing issues.